### PR TITLE
Show CREATED AT instead of AGE in deploy list

### LINKED ISSUE #343: [PROPOSAL] Show deploy completion date and time
The `deploy list` command is really useful and I'm already using it sincce v0.8.0 has come.

But I think that showing the date and time the deploy completed may be more useful, mainly in troubleshooting when the EXACT date matters and when there are multiple deploys in the same day.

This is a real output for an app of mine:

```
+----------+---------+-----+------------------------------------------+
| REVISION | CURRENT | AGE |               DESCRIPTION                |
+----------+---------+-----+------------------------------------------+
| 17       | false   | 19d | c2d00239 Bump version: 1.114.2 → 1.115.0 |
+----------+---------+-----+------------------------------------------+
| 18       | false   | 12d | c00332b Bump version: 1.115.0 → 1.115.1  |
+----------+---------+-----+------------------------------------------+
| 19       | false   | 12d | 226868ea Bump version: 1.115.1 → 1.116.0 |
+----------+---------+-----+------------------------------------------+
| 20       | false   | 10d | 5290ab57 Bump version: 1.116.0 → 1.117.0 |
+----------+---------+-----+------------------------------------------+
| 21       | false   | 5d  | v1.118.0                                 |
+----------+---------+-----+------------------------------------------+
| 22       | true    | 4d  | 48f4553 Bump version: 1.118.0 → 1.119.0  |
+----------+---------+-----+------------------------------------------+
```

It would be nice if it is something like this (the `COMPLETED AT` column):

```
| ---------- | --------- | ---------------------| ------------------------------------------ |  
| REVISION   | CURRENT   | COMPLETED AT         | DESCRIPTION                                |  
| ---------- | --------- | -------------------- | ------------------------------------------ |  
| 17         | false     | 2017-09-01 01:00:00Z | c2d00239 Bump version: 1.114.2 → 1.115.0   |  
| ---------- | --------- | -----                | ------------------------------------------ |  
| 18         | false     | 2017-09-08 01:00:00Z | c00332b Bump version: 1.115.0 → 1.115.1    |  
| ---------- | --------- | -----                | ------------------------------------------ |  
| 19         | false     | 2017-09-08 01:00:00Z | 226868ea Bump version: 1.115.1 → 1.116.0   |  
| ---------- | --------- | -----                | ------------------------------------------ |  
| 20         | false     | 2017-09-10 01:00:00Z | 5290ab57 Bump version: 1.116.0 → 1.117.0   |  
| ---------- | --------- | -----                | ------------------------------------------ |  
| 21         | false     | 2017-09-14 01:00:00Z | v1.118.0                                   |  
| ---------- | --------- | -----                | ------------------------------------------ |  
| 22         | true      | 2017-09-15 01:00:00Z | 48f4553 Bump version: 1.118.0 → 1.119.0    |  
| ---------- | --------- | -------------------- | ------------------------------------------ |  

```

(I've changed the tabke format a bit, but it's just for formatting purpose)

The example is in iso format, but it may be formatted with local computer time.

### PR DESCRIPTION
**Pull request by yagonobre**
Fixes #343

### CODE REVIEW

File: pkg/protobuf/deploy/deploy.proto [hunk 1]
```diff
@@ -37,8 +37,9 @@ message ListResponse {
     message Deploy {
         string revision = 1;
         string description = 2;
-        int64 age  = 3;
+        reserved 3;
```
Comment (drgarcia1986) [inline pkg/protobuf/deploy/deploy.proto]: mixed fellings

Comment (yagonobre) [reply pkg/protobuf/deploy/deploy.proto]: Another option is rename to `OBSOLETE_age` but not seems good =/

**Comment by drgarcia1986** [comment]
Could fix the CHANGELOG ?

**Comment by yagonobre** [comment]
@aguerra @drgarcia1986 updated

**Pull request merged**